src/dsql/parse.y

```diff
@@ -9749,10 +9753,12 @@ non_reserved_word
 	// added in FB 4.0.2
 	| BLOB_APPEND
 	// added in FB 5.0
+	| BLOBID
 	| LOCKED
 	| OPTIMIZE
 	| QUARTER
 	| TARGET
+	| TEMP
```

Comment:
Its added in FB 6.0

Reply:
Do we need this feature in FB5? Considering that it was initially made to optimize the database restore in the first place. Without MON$TRANSACTION changes, of course.

Reply:
I would like to have it in FB5.

Reply:
Then it's better to leave them in FB5 section, I think. I will fix other places where I clearly chose a wrong section.